src/provision/terraform/steps/terraform.steps.yaml

```diff
@@ -0,0 +1,75 @@
+# Steps to provision an environment with Terraform
+parameters:
+  - name: terraformDirectory
```

Comment:
can we call this `provision.steps.yaml` or similar? IMO "terraform" is too generic

Reply:
Good idea, have renamed